src/OrchardCore/OrchardCore.Localization.Core/Data/DataResourceManager.cs

```diff
@@ -0,0 +1,116 @@
+using System;
```

Comment:
Where is it used? Found it in tests only

Reply:
Seems I forgot to update some files from OCC

Reply:
I use it for `ArgumentException` & `Lazy`